spec/queries/my_facilities/blood_pressure_control_query_spec.rb

```diff
@@ -87,27 +105,29 @@
         end
       end
 
-      describe '#cohort_registrations' do
-        specify do
-          expect(query.cohort_registrations.count).to eq(7)
+      context 'considers only htn diagnosed patients' do
+        describe '#cohort_registrations' do
+          specify do
+            expect(query.cohort_registrations.count).to eq(7)
```

Comment:
Again, the counts here are iffy. Of course, this spec has always been like this, so I'll let you decide whether you wanna take a stab at updating some of these.